python/features/shapes/shapes.aac

```diff
@@ -0,0 +1,79 @@
+schema:
+    name: Shape
+    package: aac.shapes
+    modifiers:
+        - abstract
+    fields:
+        - name: area
+          type: number
+        - name: perimeter
+          type: number
+---
+schema:
+    name: Triangle
+    package: aac.shapes
+    extends:
+        - name: Shape
+          package: aac.shapes
+    fields:
+        - name: side_one_length
+          type: number
+        - name: side_two_length
+          type: number
+        - name: side_three_length
+          type: number
+        - name: height
+          type: number
+---
+schema:
+    name: Circle
+    package: aac.shapes
+    extends:
+        - name: Shape
+          package: aac.shapes
+    fields:
+        - name: radius
+          type: number
+---
+schema:
+    name: Rectangle
+    package: aac.shapes
+    extends:
+        - name: Shape
+          package: aac.shapes
+    fields:
+        - name: length
+          type: number
+        - name: height
+          type: number
+---
+schema:
```

Comment:
You could use your `Square` definition to demonstrate a logical `final` modifier. That was also part of why we wanted to use the classic shape inheritance example because there are actual "ends" to inheriting shape attributes.